tools/picard.wdl

```diff
@@ -150,15 +150,6 @@ task validate_bam {
             common: true
         }
         outfile_name: "Name for the ValidateSamFile report file"
```

Comment:
I was thinking we'd leave the options exposed, but change the default to `STRICT`.

Reply:
@adthrasher I just added the option back, _but_ I set the default to `LENIENT`. I think the most reasonable default behavior for a Validate task is to *log every potential problem but continue whenever possible*. I also don't think the specifics of this param are significant in the vast majority of cases.